README.md

```diff
@@ -1,9 +1,17 @@
 # whitehall
 
+### Running the Application
+
+**Use GOV.UK Docker to run any commands that follow.**
+
 Whitehall is deployed in two modes:
 
-- 'admin' for publishers to create and manage content; and
-- 'frontend' for rendering some content under https://www.gov.uk/government and https://www.gov.uk/world.
+- 'admin' for publishers to create and manage content (e.g. <http://whitehall-admin.dev.gov.uk/government/admin/news/new>) and
```

Comment:
Nitpick time: I know you didn't add this but I have a personal vendetta against that "and". It's unnecessary in a list. Would you mind getting rid of it?

Reply:
It's still there!

Reply:
🤔  Ah there was two "and"s
I've removed on both lines and also rewritten that line a little 

```- 'frontend' for rendering some content under https://www.gov.uk/government and https://www.gov.uk/world.```

`/government` doesn't appear to be an active URL (it directs to the home page), also the `some` felt redundant.

```- 'frontend' for rendering content (eg: <http://www.gov.uk/government/get-involved>, <https://www.gov.uk/world>)```